build/wikia/transformers/transformWarframe.mjs

```diff
@@ -1,8 +1,7 @@
-'use strict';
+import getColors from 'get-image-colors';
+import { image as _image } from 'image-downloader';
```

Comment:
:-1: don't use dangling underscores out front
```suggestion
import { image as getImage } from 'image-downloader';
```

Reply:
it seems it was the vscode that did that i didn't realize